### DISCUSSION TITLE
Playlist Window Costomization

### DISCUSSION THREAD

**Discussion by hobertuefer** [hydrogen-music/hydrogen#1818] [Hydrogen on Windows]
How can I customize the playlist window - 
e.g. 
- windowsize size
- only show title without path to directory

**Comment by theGreatWhiteShark** [comment]
> windowsize size

You are right. The window sizes are fixed. I'll change that to make it resizable and let Hydrogen remember the geometry and position used last.

> only show title without path to directory

This is not supported. But you can drag the headers of the table within the Playlist window to enlarge the width of the song list column.

**Reply by hobertuefer** [reply]
would be helpfull if you could fix resizability - great, thx - i am using the win 64 version - i cannot drag the headers - in one of the previous versions this was possible -

**Reply by theGreatWhiteShark** [reply]
> But you can drag the headers of the table within the Playlist window to enlarge the width of the song list column.

> i am using the win 64 version

Oh, I missed the "Hydrogen on Windows" category. On Windows there is - and was - just a single column in the Playlist Editor. (On Linux and macOS an arbitrary bash script can be executed when switching to a song). So, having no headers to drag is expected.

I prepared you the latest version including the patch https://ci.appveyor.com/api/buildjobs/gcpgxte9kvaychkv/artifacts/Hydrogen-1.2.1-win64.exe . Could you check whether this works properly on your end?

**Reply by hobertuefer** [reply]
many thx for your friendly efforts - but, unfortunately with the installation by the link above it does not change anything - seems to be the originally release 1.2.1 version

**Reply by theGreatWhiteShark** [reply]
> seems to be the originally release 1.2.1 version

Is the version exactly `1.2.1` or something like `1.2.1-abcdfe1`?

**Reply by hobertuefer** [reply]
I cannot find any Extension to 1.2.1

**Reply by hobertuefer** [reply]
Sorry - it works - you First have to manually uninstall the current Version an then install the new one - Great - it saves the Windows size - realy what I was looking for many thx
any the way listen once to our "Hydrogen Drummer" at www.voxxboxx.ch - we are happily using it live for many years now -
